ledger/ledger-api-common/src/test/suite/scala/com/digitalasset/platform/server/api/validation/ErrorFactoriesSpec.scala

```diff
@@ -102,6 +124,13 @@ class ErrorFactoriesSpec
           v2_message =
             s"An error occurred. Please contact the operator and inquire about the request $originalCorrelationId",
           v2_details = Seq[ErrorDetails.ErrorDetail](expectedCorrelationIdRequestInfo),
+          v2_logEntry = ExpectedLogEntry(
+            Level.ERROR,
+            "LEDGER_API_INTERNAL_ERROR(4,cor-id-1): some message: Exception: message123",
+            Some(
+              "\\{err-context: \"\\{throwableO=Some\\(java.lang.Exception: message123\\), location=ErrorFactories.scala:\\d+\\}\"\\}"
```

Comment:
Could you construct this matching string in a more DRY way? All those strings are almost exactly the same and small modification of the format will invalidate them all.

Reply:
Ok, done